e2e-tests/pages/authPage.ts

```diff
@@ -46,7 +46,7 @@ export class AuthPage {
   async enterVerificationCode(code: string) {
     await this.page
       .getByText("Enter confirmation code")
-      .waitFor({ state: "attached", timeout: 3000 });
+      .waitFor({ state: "attached", timeout: 5000 });
```

Comment:
Would be nice to have a variable for these, maybe a `long_timeout` and a `short_timeout`.

Reply:
I updated the whole test suite to use a new `TIMEOUTS` constants variable.